### ISSUE TITLE
Hide Chrome window?

### ISSUE THREAD

**Issue by cutzenfriend** [rpotter12/whatsapp-play#295]
Is there a possibility to hide the chrome windows while chatting?

**Comment by xandao6** [comment]
Yes, I have been working on it. I am waiting for the project to become more mature to release.

**Comment by rpotter12** [comment]
May be it can be done. If you like to work on it please go through https://miyakogi.github.io/pyppeteer/reference.html :)

**Comment by soumyashaw** [comment]
Did anyone take up this issue? If not I would like to

**Comment by rpotter12** [comment]
@soumyashaw Start working on it :)

**Comment by ritwikchakraborty123** [comment]
Sir I want to contribute

**Comment by rpotter12** [comment]
Sure @ritwikchakraborty123, Go ahead.

**Comment by ritwikchakraborty123** [comment]
Sir I am totally new here may be it will take some time with setup and all...sir @rpotter12  is there
Channel for discussion ?

**Comment by rpotter12** [comment]
Hi @ritwikchakraborty123, sorry for the late reply. Yes we have a discussion channel. You can find that in Readme's Badge section. There is a gitter badge. You can join our discussion channel just by clicking on it. :)

**Comment by Amulya310** [comment]
Hey, I would really like to work on this issue under GSSOC'21 .Kindly consider assigning this to me!!

**Comment by rpotter12** [comment]
Hi @Amulya310, We are not participating in GSSOC21. If you would like to work on this issue you can start working on this. :)

**Comment by dsg1320** [comment]
Hai, I would like to work on this issue under GSSOC'21. Do assign it to me

**Comment by 4NUR46** [comment]
> Is there a possibility to hide the chrome windows while chatting?
I do not have run this code but I want to tell you that,
Yes, There's a way to hide chrome window, But how can you give response to the user of their ?
If you want then you can use chrome in headless mode but for doing that you have to make some changes in program like print the received message on terminal and ask an input to give response as well. By the way I have made a bot using python and which was working well in headless mode but I have stopped working in doing modifications and made private :)
